tink/api/v1alpha1/doc.go

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Why do we need `doc.go` file if it contains no doc? :wink: Perhaps we should move the docs from `groupversion_info.go`.

Reply:
We don't necessarily, but we've had issues with web-based tools like godoc in the past with the lack of doc.go files when someone tries to access docs specific to the api types and the project wasn't already indexed

Reply:
Godoc require at least one file in the package to have documentation, it doesn't necessarily be `doc.go` file. Though when you have many files in the package, IMO it's good to have a separate file for docs, so no other files becomes "special".

TL;DR I suggest moving the `Package v1alpha1 ..` comment from `groupversion_info.go` to `doc.go` file.